tests/Doctrine/Tests/DBAL/Schema/ForeignKeyConstraintTest.php

```diff
@@ -8,6 +8,20 @@
 
 class ForeignKeyConstraintTest extends TestCase
 {
+    /**
+     * @param string[] $indexColumns
+     *
+     * @dataProvider getIntersectsIndexColumnsData
+     */
+    public function testIntersectsIndexEscapedColumns(array $indexColumns, $expectedResult)
```

Comment:
What's `$expectedResult`? Needs type documentation here

Reply:
I guess I could add bool type hint here. Is that would be ok? or I have to update doc block?

Reply:
Adding a type declaration would be best (instead of docblock)